### PR TITLE
refs #10970 - fixing some actions test failures

### PR DESCRIPTION
**Pull request by komidore64**

### CODE REVIEW

**Comment by komidore64** [comment]
plz2review @jlsherrill

File: test/actions/pulp/repository/sync_test.rb [hunk 1]
```diff
@@ -11,10 +11,6 @@ class VCRTestBase < TestBase
     include VCR::TestCase
     let(:repo) { katello_repositories(:fedora_17_x86_64) }
 
-    def run_action(action_class, *args)
```
Comment (cpeters) [inline test/actions/pulp/repository/sync_test.rb]: Does removing this method cause failures in other tests that may depend on this?

File: test/actions/pulp/repository/upload_file_test.rb [hunk 2]
```diff
@@ -18,7 +18,7 @@ def test_upload_file
       run_action(::Actions::Pulp::Repository::DeleteUploadRequest,
                   upload_id: upload_request.output[:upload_id])
 
-      assert_equal repo.puppet_module_ids.length, 1
+      assert_equal 1, repo.pulp_puppet_module_ids.length
```
Comment (cpeters) [inline test/actions/pulp/repository/upload_file_test.rb]: Good catch on the previous test have expected and actual reversed!

File: test/actions/pulp/repository/sync_test.rb (hunk 1 above)
Comment (komidore64) [inline test/actions/pulp/repository/sync_test.rb]: most files seem to implement their own `VCRTestBase` so I don't think so

File: test/actions/pulp/repository/upload_file_test.rb (hunk 2 above)
Comment (komidore64) [inline test/actions/pulp/repository/upload_file_test.rb]: :+1:

**Comment by cpeters** [comment]
ACK

**Pull request merged**